src/f_finale.c

```diff
@@ -714,7 +714,8 @@ void F_BunnyScroll (void)
 
   if (SHORT(p2->width) == SCREENWIDTH)
   {
-    V_FillRect(0, 0, video.unscaledw, SCREENHEIGHT, v_darkest_color);
+    V_FillRect(0, 0, video.deltaw, SCREENHEIGHT, v_darkest_color);
+    V_FillRect(SCREENWIDTH + video.deltaw, 0, video.deltaw, SCREENHEIGHT, v_darkest_color);
```

Comment:
This has issue with "garbage column" if `correct_aspect_ratio == 0`. I wanted to push this to master:
```c
    V_FillRect(0, 0, video.deltaw, SCREENHEIGHT, v_darkest_color);
    V_FillRect(video.deltaw + SCREENWIDTH, 0,
               video.unscaledw - (video.deltaw + SCREENWIDTH), SCREENHEIGHT,
               v_darkest_color);
```

Reply:
Okay!